### PR TITLE
Create clients list modal

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Modified the getAll function in jobsModel to include the list of clients along with each job.
- Added a function in clientsModel to retrieve all of the clients attending a specific class.
- Made clients modal and integrated with the application.

### Testing
How did you confirm your changes worked? 
- Tested the modal separate from the app using HTML.
- Integrated the modal into the app and opened it in browser to confirm that the modal looked fine and correctly reflected information in the database.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Clients Modal |
| --- |
| Screen Shot 2021-04-21 at 12 42 33 AM |

### CODE REVIEW

File: app/public/javascripts/uiheader.js [hunk 1]
```diff
@@ -25,3 +25,7 @@ $(document).ready(function() {
 $(document).ready(function(){
     $('select').formSelect();
 });
+
+$(document).ready(function () {
```
Comment (eliasfang) [inline app/public/javascripts/uiheader.js]: I think you can technically just combine these two together right?

**Review by eliasfang — CHANGES_REQUESTED** [review]
This looks so good! Just a quick note about the images that you're importing. I don't if you knew but materialize actually have icons that I used on mine so I didn't need to import any images. For example, I know that materialize has the notifications icons, the locker icon, and the people icon. But it's up to you if you want to use those instead. You can find them here: https://materializecss.com/icons.html

File: app/views/components/clients.ejs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <%=ind + 1%>.
+            </td>
+            <td class="td-attendee-content">
+                <table style="border-collapse: initial;">
+                    <tr>
+                        <td class="td-attendee-info">
+                            <%=client.first_name%> <%=client.last_name%></br>
+                            <%=client.email%>
+                        </td>
+                        <% if(client.is_recipient == true){ %>
```
Comment (mingkaili) [inline app/views/components/clients.ejs]: You can change this expression to boolean? (<true condition>):(<false condition>)

**Review by mingkaili — CHANGES_REQUESTED** [review]

**Review by eliasfang — APPROVED** [review]
Everything looks great! I just merged my branch to main so just make sure you merge that into yours and make any necessary changes :)

**Review by mingkaili — APPROVED** [review]

**Pull request merged**